CMakeLists.txt

```diff
@@ -322,6 +322,8 @@ else ()
   message(FATAL_ERROR "Unsupported compiler: ${CMAKE_CXX_COMPILER_ID}")
 endif ()
 
+option(ENABLE_ADDRESS_SANITIZER
+       "Insert pointer and reference checks into the generated binaries" OFF)
```

Comment:
Just curious? What does this unrelated change do?

Reply:
It just exposes an existing option.